stable/atlantis/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "v0.6.0"
+appVersion: "v0.7.1"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 2.0.1
+version: 2.0.2
```

Comment:
bump to 3.0 instead as this is a breaking change

Reply:
Bumped to 3.0.0